kolibri/plugins/safe_html5_viewer/assets/src/views/SafeHtml5RendererIndex.vue

```diff
@@ -6,14 +6,19 @@
       :delay="false"
       class="loader"
     />
-    <SafeHTML
+    <div
       v-else
-      :html="html"
-      :styleOverrides="{
-        windowSizeClass: windowSizeClass,
-      }"
-      @expand-img="openLightbox"
-    />
+      class="safe-html-wrapper"
+      tabindex="0"
+      aria-label="Article content"
```

Comment:
Let's pass a translated string instead here i.e :aria-label="$tr('....')" should suffice.